melange/init

```diff
@@ -0,0 +1,76 @@
+#!/bin/busybox sh
+set -e
+set -x
+
+/bin/mount -t proc proc -o nodev,nosuid,hidepid=2 /proc
+/bin/mount -t devtmpfs -o nosuid,noexec devtmpfs /dev
+/bin/mount -t sysfs sys -o nodev,nosuid,noexec /sys
+/bin/mount -t tmpfs -o nodev,nosuid,noexec tmpfs /tmp
+
+# Setup tty/pty
+/bin/mkdir /dev/pts
+/bin/mount -t devpts devpts -o noexec,nosuid,newinstance,ptmxmode=0666,mode=0620,gid=tty /dev/pts/
+/bin/mount --bind /dev/pts/ptmx /dev/ptmx
+
+# ldconfig is run to prime ld.so.cache for glibc packages which require it.
+if [ -x /sbin/ldconfig ]; then
+	/sbin/ldconfig /lib /usr/lib /usr/lib64 || :
+fi
+
+# Fetch settings from cmdline
+DNS="$(tr ' ' '\n' < /proc/cmdline | /bin/grep "dns=" | /usr/bin/cut -d'=' -f2-)"
+HOSTNAME="$(tr ' ' '\n' < /proc/cmdline | /bin/grep "hostname=" | /usr/bin/cut -d'=' -f2-)"
+SSHKEY="$(tr ' ' '\n' < /proc/cmdline | /bin/grep "sshkey=" | /usr/bin/cut -d'=' -f2- | /bin/base64 -d)"
+
+if [ -z "${SSHKEY}" ]; then
+	/bin/echo "Missing default mount for ssh keys"
+	exit 1
+fi
+
+# We can ignore this fail, if we use a kernel with kvm_guest.config, we won't need this
+# and network will work anyway
+# If this fails and we won't have network, the ifconfig command will fail anyway.
+# Also we load cpu accelleration drivers in case those are needed.
+/usr/sbin/depmod -a || :
+/usr/bin/sort -u \
+	/sys/devices/system/cpu/modalias  \
+	/sys/devices/pci*/*/virtio*/modalias | /usr/bin/xargs -n1 /usr/sbin/modprobe 2>/dev/null || :
+# modprobe 9p if absent
+/bin/grep -q 9p /proc/filesystems || /usr/sbin/modprobe 9p
+
+# Setup default mountpoint for 9p shared dir
+/bin/mount -t 9p -o trans=virtio defaultshare /mnt/ -oversion=9p2000.L
```

Comment:
this is a weird combination of '-o key=value' and '-okey=value'.

```
mount -t 9p -otrans=virtio,version=9p2000.L defaultshare /mnt
```

or

```
mount -t 9p -otrans=virtio -oversion=9p2000.L defaultshare /mnt
```

Otherwise I'm going to read this one day and assume that '-oversion' is somehow _not_ the same as '-o version'

Reply:
Fixed,thanks!